src/store/reducers/appReducer.js

```diff
@@ -97,6 +145,9 @@ export const redux = createReducerScenario({
   changeTheme: { type: 'APP_CHANGE_THEME', reducer: updateThemeReducer },
   registerError: { type: 'APP_REDUCER_ERROR', reducer: errorReducer },
   isFetching: { multiType: fetchableTypes, reducer: pendingReducer },
+  setLogo: { type: 'SET_LOGO', reducer: logoReducer },
```

Comment:
You don't need both of these reducers since they do the same thing. I also wouldn't use `get` since that word is more typically associated with the selectors.